ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt

```diff
@@ -8,26 +8,69 @@ org.jboss.spec.javax.jms:jboss-jms-api_2.0_spec
 org.jboss.spec.javax.transaction:jboss-transaction-api_1.3_spec
```

Comment:
the above 3 lines should be removed. 

the 4 jakarta.* api artifacts should be placed here instead. See next comment.